core/src/main/kotlin/br/com/guiabolso/events/validation/LenientEventValidator.kt

```diff
@@ -0,0 +1,80 @@
+package br.com.guiabolso.events.validation
+
+import br.com.guiabolso.events.model.RawEvent
+import br.com.guiabolso.events.model.RequestEvent
+import br.com.guiabolso.events.model.ResponseEvent
+import com.google.gson.JsonNull
+import com.google.gson.JsonObject
+import org.slf4j.LoggerFactory
+
+@Deprecated(
+        message = "Used only to validate events from applications that doesn't implements protocol fully.",
```

Comment:
`Used only to validate events of applications that don't fully implement this protocol`